### ISSUE TITLE
Focus is lost on the selected tree item after opening and closing it in the editor. (L1)

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-explorer-vscode#4025]
**Describe the bug**

After selecting an item under DataSets/USS/Jobs and opening it in the editor using keyboard navigation only, then closing the item from the editor with Ctrl + W, the focus does not return to the previously selected item in the tree.

Additionally, the screen reader does not announce or recognize the Ctrl + W action for closing the opened item in the editor.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Zowe Explorer Version:
- (Optional) Zowe CLI Version:
- (Optional) Do you have secure credentials enabled?
- (Optional) Are you using a custom credential manager?

**Additional context**
